src/main/kotlin/org/moe/idea/model/impl/MOESdkPropertiesImpl.kt

```diff
@@ -7,12 +7,14 @@ data class MOESdkPropertiesImpl(
     override val home: String,
     override val coreJar: String,
     override val platformJar: String?,
-    override val junitJar: String
+    override val junitJar: String,
+    override val graalHome: String?
 ) : MOESdkProperties, Serializable {
     constructor(input: MOESdkProperties) : this(
         home = input.home,
         coreJar = input.coreJar,
         platformJar = input.platformJar,
-        junitJar = input.junitJar
+        junitJar = input.junitJar,
+        graalHome = input.graalHome
```

Comment:
When an new property is added to an existing gradle model, you must use `readGradleModelOptionalProperty()` to read it like [here](https://github.com/multi-os-engine/moe-ide-integration/blob/48d75da6b4fc011b795d85019306ad37dab9eb70/src/main/kotlin/org/moe/idea/extension/MOEGradleProjectResolver.kt#L46), otherwise you'll get an `UnsupportedMethodException` when using an old version of gradle plugin which does not have this property:
```
INFO - .project.GradleProjectResolver - Gradle project resolve error 
org.gradle.tooling.model.UnsupportedMethodException: Unsupported method: MOESdkProperties.getTestProperty().
The version of Gradle you connect to does not support that method.
To resolve the problem you can change/upgrade the target version of Gradle you connect to.
Alternatively, you can ignore this exception and read other information from the model.
	at org.gradle.tooling.model.internal.Exceptions.unsupportedMethod(Exceptions.java:33)
	at org.gradle.tooling.internal.adapter.ProtocolToModelAdapter$InvocationHandlerImpl.invoke(ProtocolToModelAdapter.java:365)
	at com.sun.proxy.$Proxy138.getTestProperty(Unknown Source)
	at org.moe.idea.model.impl.MOESdkPropertiesImpl.<init>(MOESdkPropertiesImpl.kt:18)
	at org.moe.idea.extension.MOEGradleProjectResolver.createModule(MOEGradleProjectResolver.kt:47)
```